aten/src/ATen/native/cuda/MultiLabelMarginCriterion.cu

```diff
@@ -259,10 +259,10 @@ void multilabel_margin_loss_forward_out_cuda_template(
             using accscalar_t = at::acc_type<scalar_t, true>;
             multilabel_margin_loss_forward_kernel<scalar_t, accscalar_t>
                 <<<blocks, threads, 0, at::cuda::getCurrentCUDAStream()>>>(
-                    output_tmp.data_ptr<scalar_t>(),
-                    input_.data_ptr<scalar_t>(),
-                    target_.data_ptr<int64_t>(),
-                    is_target_.data_ptr<scalar_t>(),
+                    output_tmp.mutable_data_ptr<scalar_t>(),
+                    input_.const_data_ptr<scalar_t>(),
+                    target_.const_data_ptr<int64_t>(),
+                    is_target_.mutable_data_ptr<scalar_t>(),
```

Comment:
This looks wrong

Reply:
Yes it does look wrong, but look at line 78 of this file.